scripts/generate_cloudevent_library.dart

```diff
@@ -81,8 +88,12 @@ Future<void> generateLibraryFiles(
       final srcReferencePath =
           p.relative(normalizedPath, from: '$libFileDirectory/..');
 
-      libFiles[libraryFilePath]!
-          .writeln("export '$srcReferencePath' show ${symbols.join(', ')};");
+      // Dart export statements require forward slashes for paths, regardless of the
+      // operating system. This ensures compatibility, especially on Windows.
+      final dartSrcReferencePath =
+          srcReferencePath.replaceAll(p.separator, '/');
+      libFiles[libraryFilePath]!.writeln(
```

Comment:
Is it possible to check that `libFiles[libraryFilePath] != null` so that we can throw an error with a message related to what happening here? For example:

```dart
if (libFiles[libraryFilePath] != null) {
  throw MyException("error message");
}
// proceed to access the value
```